static/css/components/form.olform.less

```
@@ -48,7 +48,7 @@
     width: 4em;
   }
 
-  input[type=email],
+  input[type=text],
   input[type=password],
   .required {
```

```suggestion
  .required {
```
Removing the input selectors solves this issue, and doesn't resize other fields in other views.